src/main/java/seedu/duke/storage/Storage.java

```diff
@@ -168,11 +170,13 @@ public void save(ItemList items, String saveFilePath) throws DukeException {
         try {
             fw.write(saveString);
             fw.close();
+
         } catch (IOException e) {
             throw new DukeException(Messages.EXCEPTION_SAVE_FILE);
         }
     }
 
+
```

Comment:
Whitespace

Reply:
yeap noted for all, removed them already as well as the Manifest file